docs/protocol/network/overview.md

```diff
@@ -0,0 +1,52 @@
+# Witnet Network Protocol
+
+Witnet network protocol is inspired in other blockchain network protocols such as Bitcoin, Ethereum, Exonum, Mimblewimble Grin and Rchain. Additionally, other blockchains such as Grin and Exonum have been taken into consideration as their reference implementations are coded in Rust.
```

Comment:
Isn't there a redundancy here when mentioning Grin and Exonum twice?

Reply:
Ups.. it was a last minute mistake.